x-pack/test/api_integration/apis/maps/maps_telemetry.ts

```diff
@@ -30,8 +30,8 @@ export default function ({ getService }: FtrProviderContext) {
           return fieldStat.name === 'geo_point';
         }
       );
-      expect(geoPointFieldStats.count).to.be(31);
-      expect(geoPointFieldStats.index_count).to.be(9);
+      expect(geoPointFieldStats.count).to.be(39);
```

Comment:
This value had to be incremented when we enabled the new generic threshold rules: https://github.com/elastic/kibana/pull/162605/files#diff-53c98e8d9e2d61d5c9f37b003fc6de5c3f84570f7886cf4bdd89f596b32a15c9R49